packages/react-native/ReactAndroid/hermes-engine/build.gradle.kts

```diff
@@ -39,12 +39,28 @@ fun getSDKManagerPath(): String {
   val ossSdkManagerPath = File("${getSDKPath()}/tools/bin/sdkmanager")
   val windowsMetaSdkManagerPath = File("${getSDKPath()}/cmdline-tools/latest/bin/sdkmanager.bat")
   val windowsOssSdkManagerPath = File("${getSDKPath()}/tools/bin/sdkmanager.bat")
+  val easSdkManagerPath = File("${getSDKPath()}/cmdline-tools/tools/bin/sdkmanager")
```

Comment:
why `metaSdkManagerPath` (i.e. `/cmdline-tools/latest/bin/sdkmanager`) won't work here?

Reply:
That is not where sdkmanager is found in our latest Linux images

Reply:
Ok then let's rename this field `linuxSdkManagerPath`